src/icons.rs

```diff
@@ -287,9 +317,19 @@ lazy_static! {
         "" => "",
         "backlight_empty" => "\u{e38d}", // nf-weather-moon_new
         "backlight_full" => "\u{e39b}", // nf-weather-moon_full
-        "backlight_partial1" => "\u{e3d1}", // nf-weather-moon_alt_waxing_gibbous_3
-        "backlight_partial2" => "\u{e3ce}", // nf-weather-moon_alt_first_quarter
-        "backlight_partial3" => "\u{e3cb}", // nf-weather-moon_alt_waxing_crescent_4
+        "backlight_1" => "\u{e3d4}", // nf-weather-moon_alt_waxing_gibbous_6
```

Comment:
Update this list too: https://github.com/greshake/i3status-rust/blob/master/themes.md#available-icon-overrides